lib/gather/analyze-ember-object.js

```diff
@@ -10,6 +12,18 @@ module.exports = function analyzeEmberObject(possibleEmberObject) {
         type: 'Helper',
       };
     } else if (typeof eObjDefault.proto !== 'function') {
+      /**
+       * this is for JavaScript-less components that only have a template. Only
+       * components with a backing JavaScript file will be in the `require.entries`
+       * that will have a `proto()` we can instantiate.
+       */
+      if (modulePath && modulePath.includes('templates/components/')) {
```

Comment:
for pods-like structure it will `endsWith('/template')`

Reply:
@lifeart would you mind PR’ing that change? If not I’ll get to it as soon as I can

Reply:
looks like it won't work for podded routes (routes templates will be marked as components templates)